packages/sra-report/src/index.ts

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node
+import { assert } from '@0xproject/assert';
+import { Schema, schemas } from '@0xproject/json-schemas';
+import { promisify } from '@0xproject/utils';
+import chalk from 'chalk';
+import * as _ from 'lodash';
+import * as newman from 'newman';
+import * as yargs from 'yargs';
+
+import * as sraReportCollectionJSON from '../postman_configs/collections/sra_report.postman_collection.json';
+
+import { postmanEnvironmentFactory } from './postman_environment_factory';
+import { utils } from './utils';
+
+const newmanRunAsync = promisify<void>(newman.run);
+const DEFAULT_NETWORK_ID = 1;
+const SUPPORTED_NETWORK_IDS = [1, 42];
```

Comment:
Can we please add support for Rinkeby and Ropsten? We are deployed on those networks and I know for a fact that some relayers are using them as their testnets.